### PR TITLE
fix website

### PR DESCRIPTION
**Pull request by danking**
@cseed @jbloom22 

I had forgotten the website's index.html is different from the root README.md. I just copied over the README.md to here. Is there anything from the current website we want to keep? At the very least the overview link was broken.

### CODE REVIEW

File: hail/www/index.md [hunk 1]
```diff
@@ -1,63 +1,82 @@
...
-- chat with the Hail team and user community in Hail's [Zulip chatroom](https://hail.zulipchat.com).
-- please report any suspected bugs as [GitHub issues](https://github.com/hail-is/hail/issues)
+Contact the Hail team at <a href="mailto:hail@broadinstitute.org"><code>hail@broadinstitute.org</code></a>.
 
-### Maintainer
+### Citing Hail
 
-Hail is maintained by a team in the [Neale lab](https://nealelab.squarespace.com/) at the [Stanley Center for Psychiatric Research](http://www.broadinstitute.org/scientific-community/science/programs/psychiatric-disease/stanley-center-psychiatric-research/stanle) of the [Broad Institute of MIT and Harvard](http://www.broadinstitute.org) and the [Analytic and Translational Genetics Unit](https://www.atgu.mgh.harvard.edu/) of [Massachusetts General Hospital](http://www.massgeneral.org/).
+If you use Hail for published work, please cite the software. You can get a
+citation for the version of hail you installed by executing:
```
Comment (jbloom22) [inline hail/www/index.md]: [H]ail

File: hail/www/index.md [hunk 2]
```diff
@@ -1,63 +1,82 @@
...
-Hail is a **[Python](https://www.python.org/)**  library with a scalable backend built on top of **[Apache Spark](https://spark.apache.org/docs/latest/index.html)** to efficiently analyze gigabyte-scale data on a laptop or terabyte-scale data on a cluster. 
+See the [documentation](https://hail.is/docs/0.2/) for more info on using Hail.
 
-### Getting Started
+### Contribute
 
-To get started using Hail:
+If you'd like to discuss or contribute to the development of methods or infrastructure, please:
 
-- install Hail 0.2 using the instructions in [Installation](https://hail.is/docs/0.2/getting_started.html)
```
Comment (jbloom22) [inline hail/www/index.md]: I think we should keep some of these bullets after "See the documentation" to help users understand what's there and navigate to the right place. (When we split the readme from the index, we also split bullets for users and contributors there)

Comment (danking) [reply hail/www/index.md]: Con: they must click the documentation link
Pro: the links don't break

I kind of prefer not calling out these doc sections here and on the docs landing page. Thoughts?

Comment (tpoterba) [reply hail/www/index.md]: the documentation landing page is shit. we need to link somewhere other than that, or fix it

Comment (jbloom22) [reply hail/www/index.md]: I agree with Tim. In the meantime, Dan, you should at least bullet what's not in the docs: the user chat and user forum

**Comment by danking** [comment]
I've added back zulip and the forum. Punting on docs landing page.

**Pull request merged**